src/ast.js

```diff
@@ -1447,7 +1442,7 @@ function ast_for_funcdef_impl(c, n0, decorator_seq, is_async) {
     var name_i = 1;
     var end_lineno, end_col_offset;
     var tc;
-    var type_comment = NULL;
+    var type_comment = new Sk.builtin.str("");
```

Comment:
was this to placate closure?

Reply:
Yes, but this code never worked, so I don't know if any of it is correct at all.  This is just a guess at what is supposed to be here.  It probably breaks something else, but I don't know what.  Do you have any tests that actually exercise this code? There's nothing in our test suite that does.  And the few things I tried didn't actually end up going through here.

So, I probably moved this sideways.  From not working at all, to passing closure, but probably still not working at all.